core/src/config.rs

```diff
@@ -133,6 +135,28 @@ impl Configuration {
     }
 }
 
+impl Default for Configuration {
```

Comment:
We could still #[derive(Default)] by inverting the boolean: e.g. `disable_color_eyre`.